src/components/Status/Status.js

```diff
@@ -35,29 +41,64 @@ OverlayStatus.defaultProps = {
 
 OverlayStatus.propTypes = {
   icon: PropTypes.string,
-  text: PropTypes.string.isRequired,
+  children: PropTypes.oneOfType([PropTypes.node, PropTypes.func]).isRequired,
   overlay: PropTypes.oneOfType([PropTypes.node, PropTypes.func]).isRequired,
 };
 
-export const LinkStatus = ({ icon, children, linkMessage, linkTo }) =>
-  linkTo ? (
-    <Status icon={icon}>
-      <Link to={linkTo} title={linkMessage}>
-        {children}
-      </Link>
-    </Status>
+export const LinkStatus = ({ icon, children, title, information, linkMessage, linkTo, message }) => {
+  if (!(linkTo || title)) {
+    return <Status icon={icon}>{children}</Status>;
+  }
+
+  const infoSection = information ? <dd>{information}</dd> : '';
+
+  const messageSection = message ? (
+    <React.Fragment>
+      <dt>Message:</dt>
```

Comment:
I'd rather do this with `<div>`

Reply:
do we have a class for this format ? should I create a new one ?

Reply:
new one

Reply:
:+1: added a new class ...